webapp/src/pages/User/settings/EditProfileTab.tsx

```diff
@@ -48,149 +91,269 @@ function EditProfileTab() {
           <CardContent sx={{ p: 4 }}>
             <Typography variant="subtitle2">
-              <Grid container spacing={0}>
+              <Grid justifyItems={'center'} container spacing={0}>
                 <Grid item xs={12} sm={4} md={3} textAlign={{ sm: 'right' }}>
                   <Box pr={3} pb={2}>
-                    Language:
+                    Firstname:
                   </Box>
                 </Grid>
                 <Grid item xs={12} sm={8} md={9}>
                   <Text color="black">
-                    <b>English (US)</b>
+                    {!EditUser ? (
+                      <b>{user.firstname}</b>
+                    ) : (
+                      <TextField
+                        id="user-firstname"
+                        label="Firstname"
+                        name="firstname"
+                        onChange={(e: any) => {
+                          handleChangeEdit(e);
+                        }}
+                        variant="standard"
+                        size="small"
+                      />
+                    )}
                   </Text>
                 </Grid>
                 <Grid item xs={12} sm={4} md={3} textAlign={{ sm: 'right' }}>
                   <Box pr={3} pb={2}>
-                    Timezone:
+                    Lastname:
                   </Box>
                 </Grid>
                 <Grid item xs={12} sm={8} md={9}>
                   <Text color="black">
-                    <b>GMT +2</b>
+                    {!EditUser ? (
+                      <b>{user.lastname}</b>
+                    ) : (
+                      <TextField
+                        id="user-lastname"
+                        label="Lastname"
+                        name="lastname"
+                        onChange={(e: any) => {
+                          handleChangeEdit(e);
+                        }}
+                        variant="standard"
+                        size="small"
+                      />
+                    )}
                   </Text>
                 </Grid>
                 <Grid item xs={12} sm={4} md={3} textAlign={{ sm: 'right' }}>
                   <Box pr={3} pb={2}>
-                    Account status:
+                    Email:
                   </Box>
                 </Grid>
                 <Grid item xs={12} sm={8} md={9}>
-                  <Label color="success">
-                    <DoneTwoToneIcon fontSize="small" />
-                    <b>Active</b>
-                  </Label>
+                  <Text color="black">
+                    {!EditUser ? (
+                      <b>{user.email}</b>
+                    ) : (
+                      <TextField
+                        type="email"
+                        id="user-email"
+                        label="Email"
+                        name="email"
+                        onChange={(e: any) => {
+                          handleChangeEdit(e);
+                        }}
```

Comment:
```suggestion
                        onChange={handleChangeEdit}
```

je pense que ça fonctionne comme ça?